qlib/constant.py

```
@@ -19,4 +19,6 @@
 ONE_DAY = pd.Timedelta("1day")
 ONE_MIN = pd.Timedelta("1min")
 EPS_T = pd.Timedelta("1s")  # use 1 second to exclude the right interval point
-float_or_ndarray = TypeVar("float_or_ndarray", float, np.ndarray)
+float_or_ndarray = Any
+
+list = []
```

Pylint (3.3.7)

[warning] Redefining built-in 'list'

(W0622)